<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProtocol</class>
 <widget class="QDialog" name="DialogProtocol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模板配置</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>781</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,20">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="15,2,5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,10,1,3,10">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>设备模板：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>规约：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonProtoAdd">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>新增</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonProtoDele">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonProtoFresh">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>761</width>
         <height>381</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFaAdd">
            <property name="text">
             <string>新增</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFaFresh">
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
